requirements.txt

```diff
@@ -19,3 +19,4 @@ beautifulsoup4
 posthog
 sqlfluff
 ruamel.yaml
+slack_sdk
```

Comment:
not sure if needed to be added manually

Reply:
it's generated automatically but the result will be the same so it's ok :)